Tests/DependencyInjection/Compiler/AddDependencyCallsCompilerPassTest.php

```diff
@@ -612,6 +642,16 @@ private function getContainer()
             ->setClass('Sonata\AdminBundle\Tests\DependencyInjection\MockAdmin')
             ->setArguments(array('', 'Sonata\AdminBundle\Tests\DependencyInjection\Report', 'SonataAdminBundle:CRUD'))
             ->addTag('sonata.admin', array('group' => 'sonata_report_group', 'manager_type' => 'orm', 'on_top' => true));
+        $container
+            ->register('sonata_report_one_admin')
+            ->setClass('Sonata\AdminBundle\Tests\DependencyInjection\MockAdmin')
+            ->setArguments(array('', 'Sonata\AdminBundle\Tests\DependencyInjection\ReportOne', 'SonataAdminBundle:CRUD'))
+            ->addTag('sonata.admin', array('group' => 'sonata_report_one_group', 'manager_type' => 'orm', 'show_mosaic_button' => false));
```

Comment:
because the name is `testProcessMultipleOnTopOptions()`

you should create a method called `testProcessShowMosaicButtonOption` i.e.